test/screenshot/spec/mdc-button/classes/baseline-button-with-icons.html

```diff
@@ -44,12 +44,14 @@
       <div class="test-layout">
         <div class="test-cell test-cell--button">
           <button class="mdc-button">
+            <div class="mdc-button__ripple"></div>
```

Comment:
Can you also update the screenshots in spec/button/mixins folder?

Reply:
Done. Also updated screenshots for spec/dialog/mixins.